awx/main/models/workflow.py

```diff
@@ -114,6 +114,9 @@ def _get_workflow_job_field_names(cls):
             'credentials',
             'char_prompts',
             'all_parents_must_converge',
+            'labels',
```

Comment:
Are there any other objects in here that we need add fields to? WorkflowJobTemplateNode?

Reply:
We also have a failing test.